README.md

```diff
@@ -42,7 +42,7 @@ Company                                           | Business/products
 [Elastic](https://www.elastic.co/)                | Elasticsearch as a service               | Java, DevOps, Web                              | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.elastic.co/about/careers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Elastic-EI_IE751551.11,18.htm)
 [Envato](https://envato.com/)                     | Sites for creatives                      | Ruby, Web                                      | ✓                                    | Anywhere                | :keyboard: [Jobs](https://envato.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Envato-EI_IE700841.11,17.htm)
 [Eyeo](https://eyeo.com)                          | AdBlock Plus & More                      | Many                                           | ?                                    | Anywhere                | :keyboard: [Jobs](https://eyeo.com/jobs/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Eyeo-EI_IE848298.11,15.htm)
-[Fire Engine Red](http://fire-engine-red.com/)    | Education market solutions               | ?                                              | ?                                    | Anywhere                | :keyboard: [Jobs](http://fire-engine-red.com/about/#careers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Fire-Engine-RED-EI_IE1339735.11,26.htm)
+[Fire Engine Red](http://fire-engine-red.com/)    | Education market solutions               | ?                                              | ?                                    | Anywhere                | :keyboard: [Jobs](https://fire-engine-red.com/career/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Fire-Engine-RED-EI_IE1339735.11,26.htm)
```

Comment:
Hmm... Now that I look at the career page, I see that they don't hire outside the Americas, so they shouldn't be on this list. Can you please remove this row?